flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/OPTIONS/A320_Neo_CDU_OPTIONS_MCDU_KB.js

```diff
@@ -0,0 +1,84 @@
+class CDU_OPTIONS_MCDU_KB {
+    static ShowPage(mcdu) {
+
+        mcdu.clearDisplay();
+
+        const mcduInput = NXDataStore.get("MCDU_KB_INPUT", "NONE");
```

Comment:
I don't understand why you are using NONE and ALL instead of 0 and 1 to mimic Boolean behavior which you are strictly following.

Reply:
Was leftover for when there were 3 modes earlier on, reduced down to 2. I agree that it's not needed now.